### ISSUE TITLE
Mars Bulwark ALT text should be in the main section. More information about cone size / angles should be in alt text.

### ISSUE THREAD

**Issue by ds0nts2** [ValveSoftware/Dota2-Gameplay#9620]
### Hero and spell

Bulwark

### Description

The Main Text
"Mars redirects n% of projectiles aimed at Mars's allies towards Mars's instead".

The Alt Text
"Mars redirect only works if the projectile targets are in a cone behind mars or within 100 range of him"

1. Thats super important. The main text just says,  Without reading the alt text, the skill is unusable. People might get lucky and use it right. But most people would just hide in some trees and turn it on and go wtf why doesnt this work. Or not be in between the attacker and attacked

2. It doesnt tell you enough about this 'cone', which is about 70 degrees on either side behind mars. It took me like 10 minutes in demo to figure it out.

3. It doesn't tell you that the attacker needs to be in front of you by approximately the same cone, equivalently, it does not mention that mars has to face the opponent.

Thanks for reading.
